models/demos/falcon7b/tt/model_config.py

```diff
@@ -129,6 +130,55 @@ def get_model_config(model_config_str, prefill_seq_len=0):
     model_config.update({f"{key}_MEMCFG": mem_config for key in OP_KEYS if key not in NO_MEMCFG})
     model_config.update({f"{key}_DTYPE": dtype for key in OP_KEYS if key not in NO_DTYPE})
 
+    ln_max_num_cores_y = 8
+    for i in range(ln_max_num_cores_y, 0, -1):
+        if (prefill_seq_len // 32) % i == 0:
+            ln_num_cores_y = i
+            break
+
+    ln_num_cores_x = 8
+    hidden_dim = 4544
```

Comment:
This can be taken from configuration dictionary that we either get from reference model if we run it, or like this in demo:

`configuration = FalconConfig(**model_config_entries)`

Reply:
done